index.php

```diff
@@ -19,24 +33,40 @@
 	<div class="top left">
 		<div class="date small dimmed"></div>
 		<div class="time"></div>
-		<div class="timeWarning xxsmall"></div>
-		<div class="holidays xxsmall dimmed"></div>
-		<div class="calendar xxsmall"></div>
```

Comment:
Calendar doesn't need to be removed; simply not having a calendar will keep it from showing up.

Similarly, if you don't define the MBTA block, it won't be displayed.